src/Symfony/Component/Security/Core/Validator/Constraints/UserPassword.php

```diff
@@ -29,8 +32,32 @@ class UserPassword extends Constraint
     public $message = 'This value should be the user\'s current password.';
     public $service = 'security.validator.user_password';
 
-    public function __construct(array $options = null, string $message = null, string $service = null, array $groups = null, mixed $payload = null)
-    {
+    public function __construct(
+        /*
+         * Options to pass the validation constraint.
```

Comment:
This reads a bit confusing ... maybe: _"Options passed to the validation constraint"_